src/Hacknet/ui/HacknetServerElem.tsx

```diff
@@ -232,9 +250,29 @@ export function HacknetServerElem(props: IProps): React.ReactElement {
               <Typography>Production:</Typography>
             </TableCell>
             <TableCell colSpan={2}>
-              <Typography>
-                <Hashes hashes={node.totalHashesGenerated} /> (<HashRate hashes={node.hashRate} />)
-              </Typography>
+              <Tooltip
+                title={
+                  <Typography>
+                    <Hashes hashes={node.totalHashesGenerated} /> hashes producted by this server since last augment
+                    installation.
+                    <br />
+                    <HashRate hashes={node.hashRate} /> current production rate.
+                    <br />
+                    <HashRate hashes={(node.hashRate * node.maxRam) / (node.maxRam - node.ramUsed)} /> max production
+                    rate. (achieved when 100% Ram is allocated to it)
+                    <br />
+                    {numeralWrapper.formatRAM(node.ramUsed)} / {numeralWrapper.formatRAM(node.maxRam)} (
+                    {Math.round((100 * node.ramUsed) / node.maxRam)}%) Ram allocated to script.
+                    <br />
+                    {numeralWrapper.formatRAM(node.maxRam - node.ramUsed)} / {numeralWrapper.formatRAM(node.maxRam)} (
+                    {Math.round((100 * (node.maxRam - node.ramUsed)) / node.maxRam)}%) Ram allocated to hash production.
+                  </Typography>
```

Comment:
```suggestion
                  <Typography>
                    <Hashes hashes={node.totalHashesGenerated} /> hashes produced by this server since last augment
                    installation.
                    <br />
                    <HashRate hashes={node.hashRate} /> current production rate.
                    <br />
                    <span style={{ opacity: 0.75 }}>
                      <HashRate hashes={(node.hashRate * node.maxRam) / (node.maxRam - node.ramUsed)} /> max production
                    rate. (achieved when 100% RAM is allocated to it)
                    </span>
                    <br />
                    {numeralWrapper.formatRAM(node.ramUsed)} / {numeralWrapper.formatRAM(node.maxRam)} (
                    {Math.round((100 * node.ramUsed) / node.maxRam)}%) RAM allocated to script.
                    <br />
                    {numeralWrapper.formatRAM(node.maxRam - node.ramUsed)} / {numeralWrapper.formatRAM(node.maxRam)} (
                    {Math.round((100 * (node.maxRam - node.ramUsed)) / node.maxRam)}%) RAM allocated to hash production.
                  </Typography>
```
typo fix, some capitalization, and an opacity thing again

Reply:
I'm settling for  : 
```
<span style={{ opacity: 0.50 }}>
  <HashRate hashes={(value)} /> 
</span>
{" "}descriptive text
```

I find it makes contrast between current and theorical value better, and keep the lengthy description that follow eyes-friendly.

![image](https://user-images.githubusercontent.com/104104269/164895279-51143726-703d-402e-9928-1e5cd4b64f22.png)


![image](https://user-images.githubusercontent.com/104104269/164895224-e0f2ff44-490c-41c9-ba8e-8bfba8ee5ae5.png)

Your inputs are deeply appreciated :-) 
I'm (very!) new to the React framework, so I have a hard time figuring what can be done, what cannot, and what can but shouldn't. :-p

Reply:
OCD at work : putting the undimmed value above dimmed one for constency sake.

![image](https://user-images.githubusercontent.com/104104269/164896145-ed415670-3fa1-4866-965a-575c1e931d58.png)

Reply:
woo that looks really good! im pretty new to react as well, so im glad to have helped :)